applications/openshift/registry/ocp_allowed_registries_for_import/rule.yml

```diff
@@ -17,3 +17,13 @@ severity: medium
 warnings:
     - general: |-
         {{{ openshift_cluster_setting("/apis/config.openshift.io/v1/images/cluster") | indent(8) }}}
+
+template:
+    name: yamlfile_value
+    vars:
+        filepath: /apis/config.openshift.io/v1/oauths/cluster
```

Comment:
This should be `/apis/config.openshift.io/v1/images/cluster`, and not `oauths`

Reply:
Good catch, thanks!